assets/js/googlesitekit/modules/datastore/modules.test.js

```diff
@@ -1983,5 +2025,65 @@ describe( 'core/modules modules', () => {
 				);
 			} );
 		} );
+
+		describe( 'getShareableModules', () => {
+			it( 'should return undefined if the call to retrieve modules fails', async () => {
+				fetchMock.getOnce(
+					new RegExp( '^/google-site-kit/v1/core/modules/data/list' ),
+					{ body: {}, status: 400 }
+				);
+
+				const initialShareableModules = registry
+					.select( CORE_MODULES )
+					.getShareableModules();
+				expect( initialShareableModules ).toBeUndefined();
+
+				await untilResolved( registry, CORE_MODULES ).getModules();
+
+				const shareableModules = registry
+					.select( CORE_MODULES )
+					.getShareableModules();
+
+				expect( console ).toHaveErrored();
+				expect( shareableModules ).toBeUndefined();
+			} );
+
+			it( 'should return an empty object if there are no internal shareable modules', async () => {
+				fetchMock.getOnce(
+					new RegExp( '^/google-site-kit/v1/core/modules/data/list' ),
+					{ body: FIXTURES, status: 200 }
+				);
+
+				const initialShareableModules = registry
+					.select( CORE_MODULES )
+					.getShareableModules();
+				expect( initialShareableModules ).toBeUndefined();
+
+				await untilResolved( registry, CORE_MODULES ).getModules();
+
+				const shareableModules = registry
+					.select( CORE_MODULES )
+					.getShareableModules();
+
+				expect( shareableModules ).toMatchObject( {} );
+			} );
+
+			it( 'should return the modules object for each internal shareable module', () => {
```

Comment:
Similar to the documentation note above, this is checking to ensure every module returned is _not_ internal (which is the correct behaviour), so this test spec should be updated.